### PR TITLE
Get birds speccies

### PR DESCRIPTION
**Pull request by thasipablo**
- Build files structure for your React app. 
- Create routes and navigation in your app.
- Display a list of items and filter them. The data come from the API.
- Style the components to align with the design provided.
- Deploy the project.
- Record a video for the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 Presentation [video](https://www.loom.com/share/fd2bba6c2fdd4d889c8eb27edaf7142d?sid=59fb050c-b999-47af-b06d-a5dde4792d95)
 
-Checkout the live version clicking on this [link](https://github.com/)
+Checkout the live version clicking on this [link](https://lovelybirds.vercel.app/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (DeliceLydia) [inline README.md:63]: - [x] Please gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Comment (DeliceLydia) [reply README.md:63]: - [x] Please add unit tests for several components of the app

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @thasipablo, 

Good job so far! There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by thasipablo** [comment]
Hi @DeliceLydia . Thanks for your review.

I gave appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) and added unit tests for some components of the app.

Thank you,

**Review by Efiamotu-1 — APPROVED** [review]
Hi @thasipablo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

image

## Highlights :dart: 
* Correct Gitflow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by thasipablo** [comment]
Thank you @Efiamotu-1

**Pull request merged**